src/Symfony/Component/Console/Attribute/AsCommand.php

```diff
@@ -25,13 +25,15 @@ class AsCommand
      * @param string[]    $aliases     The list of aliases of the command. The command will be executed when using one of them (i.e. "cache:clean")
      * @param bool        $hidden      If true, the command won't be shown when listing all the available commands, but it can still be run as any other command
      * @param string|null $help        The help content of the command, displayed with the help page
+     * @param string[]    $usages      The list of usage examples, displayed with the help page.
      */
     public function __construct(
         public string $name,
         public ?string $description = null,
         array $aliases = [],
         bool $hidden = false,
         public ?string $help = null,
+        public array $usages = [],
```

Comment:
can we also update the `AddConsoleCommandPass` to configure it via service tag?

Reply:
Makes sense. If anyone else cares to do that, feel free. If not I will figure that out tomorrow. Is there a test that I could adjust to show that its working?